coalib/io/FileFactory.py

```diff
@@ -0,0 +1,127 @@
+import os
+
+from coala_utils.decorators import generate_eq
+
+
+@generate_eq('name', 'timestamp')
+class FileFactory:
+    """
+    The ``FileFactory`` is a wrapper class to replace the actual file contents
+    from the ``file-dict``. The object contains the following information about
+    the file:
+
+        * The filename (absolute file path).
+        * The modification timestamp of the file.
+        * The file in the form of a string.
+        * The file (UTF-8 decoded) in the form of a tuple.
```

Comment:
you should say that these are all lines inside the tuple (not that people think each char :sweat_smile: )